services/subscriber/http.go

```diff
@@ -1,39 +1,88 @@
 package subscriber
 
 import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
 	"time"
 
-	"github.com/influxdata/influxdb/client/v2"
 	"github.com/influxdata/influxdb/coordinator"
 )
 
 // HTTP supports writing points over HTTP using the line protocol.
 type HTTP struct {
-	c client.Client
+	url         *url.URL
+	accessToken string
+	httpClient  *http.Client
```

Comment:
Is there likely to be any other type of client in the `HTTP` type? Maybe `client`, or even `c` is a better name.